freenet-ext/build.xml

```diff
@@ -251,22 +256,27 @@
 
 	<macrodef name="lib-report">
 		<attribute name="lib" />
-		<sequential>
-			<if>
-				<istrue value="${extlib.@{lib}.suppress}"/>
-				<then><echo message="suppress @{lib} is ON"/></then>
-			</if>
+		<sequential if:true="${extlib.@{lib}.suppress}">
+			<echo message="suppress @{lib} is ON"/>
+		</sequential>
+		<sequential unless:true="${extlib.@{lib}.suppress}">
+			<!-- To preserve the old behaviour of this macro, I've commented this out.
+			<echo message="suppress @{lib} is NOT ON"/>
```

Comment:
I also want to point out that having `<echo />` when `extlib.${lib}.suppress` is not `true` changes the behavior of the *lib-report* `<macrodef>` involved. It is commented out because this commit was meant to preserve the original behavior of the build script. When I squash these commits together, I'll remove the comment entirely.
If we're going to change the behavior of the build script, it should be done in a separate commit.